concepts.wip/booleans/introduction.md

```diff
@@ -1,14 +1,17 @@
 # Introduction
 
+## Booleans in Julia
+
 True or false values are represented by the `Bool` type.
 It contains only two values: `true` and `false`.
 
-## Booleans in Julia
-
 ```julia
```

Comment:
```suggestion
```julia-repl
```

Reply:
Good suggestion. I didn't create this, I just copied it from `concept.wip` without knowing about `julia-rel`.

Reply:
This could ripple outwards. I guess all new content should use `julia-repl`. Should all the drafts in `exercises/concepts.wip` be upgraded? A bit tedious, but the sort of thing a newbie like me could do.

Reply:
All of the examples that are supposed to look like REPL sessions, yes. If you've got the energy to do a PR on this, then that would be welcome!

Reply:
> A bit tedious, but the sort of thing a newbie like me could do.

There's probably a way to script it by detecting the `julia>` prompt

Reply:
````
perl -pe '```julia\njulia>/```julia-repl\njulia>' filename
````

Would probably work. Just apply that to all the markdown files in the repo with something like `for filename in (git-ls-files | grep '.md$'; do`

Reply:
Done.